From a1bed925d65503bc54c46a94a73c198e2b487286 Mon Sep 17 00:00:00 2001
From: Konrad Leszczynski <konrad.leszczynski@intel.com>
Date: Wed, 9 Dec 2015 12:39:43 +0000
Subject: [PATCH 1/4] usb: xhci: remove unnecessary condition asserting MINF
 method

This patch removes the an unnecessary condition, when asserting
the ACPI method MINF. Without it, the method is asserted
correctly and the ACPI values can be read.

Change-Id: Ic18e74e4505f8b492f758a426bba98af42c05d06
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-11820
Signed-off-by: Konrad Leszczynski <konrad.leszczynski@intel.com>
Reviewed-on: https://android.intel.com:443/446226
---
 drivers/usb/host/xhci-intel-cap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/host/xhci-intel-cap.c b/drivers/usb/host/xhci-intel-cap.c
index cada8c66..7b6faab 100644
--- a/drivers/usb/host/xhci-intel-cap.c
+++ b/drivers/usb/host/xhci-intel-cap.c
@@ -73,7 +73,7 @@ int xhci_intel_need_disable_stall(struct xhci_hcd *xhci)
 	/* Get modem status from MINF function */
 	modem_status = acpi_evaluate_object(acpi_dev->handle, "MINF",
 			NULL, &buffer);
-	if (ACPI_FAILURE(modem_status) || modem_status != AE_NOT_FOUND) {
+	if (ACPI_FAILURE(modem_status)) {
 		dev_dbg(&pdev->dev, "No MINF method!\n");
 		return 0;
 	}
-- 
1.9.1

